eng/jacoco-test-coverage/pom.xml

```diff
@@ -259,7 +259,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-internal-avro</artifactId>
-      <version>12.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-internal-avro;current} -->
+      <version>12.0.2-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-internal-avro;current} -->
```

Comment:
The beta series shouls be `12.1.0 ...`

Reply:
I think this is fine. We added no features to internal avro - Just updated the dependency on azure-storage-common